### DISCUSSION TITLE
How to hide the create button on the create page

### DISCUSSION THREAD

**Discussion by khoait03** [filamentphp/filament#15035] [Help]
### Package

Form builder

### Package Version

v3.4.18

### How can we help you?

How to hide the create button on the create page 
I use canCreate and it redirects to 403 page, but I just want to hide the create button so the create page is still visible

```
public static function canCreate(): bool
{
        return false;
 }
```

**Comment by leandrocfe** [comment]
You can use this in the CreatePage

```php
protected function getCreateFormAction(): Action
{
    return parent::getCreateFormAction()
        ->hidden();
}
```

**Reply by khoait03** [reply]
Thank you very much I was successful
